test/utils/getUrl.js

```diff
@@ -18,6 +18,14 @@ var url = require('url');
 *   }
 * @return {string}
 */
-module.exports = function getPage(config, app) {
+
+module.exports = getUrl;
+
+function getUrl(config, app) {
   return url.format(_.assign(config, app));
 };
+
+getUrl.noAuth = function getUrlNoAuth(config, app) {
+  var configUrl = _.pick(config, function (val, param) { return param !== 'auth'; });
+  return url.format(_.assign(configUrl, app));
```

Comment:
Since we know that we want noAuth to produce the same results as getUrl only without the auth, I think we should actually use getUrl to handle url creation.  This way if someone could modify the getUrl implementation in the future without having to know that they should also update noAuth.  It's a bit more DRY.  Perhaps something like:

```javascript
getUrl.noAuth = function getUrlNoAuth(config, app) {
  config = _.pick(config, function (val, param) {
    return param !== 'auth';
  });
  return getUrl(config, app);
};
```